src/app/components/top/solicitudes/solicitudes.component.ts

```diff
@@ -117,7 +117,7 @@ export class SolicitudesComponent implements OnInit {
     ngOnInit() {
         if (!this.auth.getPermissions('solicitudes:?').length) {
             this.router.navigate([this.auth.profesional ? '/solicitudes/asignadas' : 'inicio']);
-        } else if (this.auth.getPermissions('solicitudes:?').length === 1 && this.auth.getPermissions('solicitudes:reglas:?')[0] !== '*') {
+        } else if (this.auth.getPermissions('solicitudes:?').length === 1 && this.auth.getPermissions('solicitudes:reglas:?')[0] !== '*' && !this.auth.getPermissions('solicitudes:tipoPrestacion:?')) {
```

Comment:
Me parece que la evaluación del `getPermissions` deberia ser por el length del array,  caso contrario a la evaluación del array vacío siempre devolverá true
```suggestion
        } else if (this.auth.getPermissions('solicitudes:?').length === 1 && this.auth.getPermissions('solicitudes:reglas:?')[0] !== '*' && !this.auth.getPermissions('solicitudes:tipoPrestacion:?').length) {
```

Reply:
@plammel Corregido